src/Core/src/Platform/iOS/MauiScrollView.cs

```diff
@@ -64,7 +64,7 @@ public override void LayoutSubviews()
 						crossPlatformLayout.CrossPlatformArrange(new Rect(new Point(-horizontalOffset, 0), crossPlatformBounds));
 						ContentOffset = new CGPoint(horizontalOffset, 0);
 					}
-					else
+					else if (_previousEffectiveUserInterfaceLayoutDirection is not null)
```

Comment:
@mjo151 If you try this, could you share a repro sample? Would like to include a related test.